commercetools/internal-docs/src/test/java/example/HelperMethodsTest.java

```diff
@@ -0,0 +1,2115 @@
+import com.commercetools.api.models.quote_request.QuoteRequestSetCustomFieldAction;
+import com.commercetools.api.models.quote_request.QuoteRequestUpdateBuilder;
+import com.commercetools.api.models.review.*;
+import com.commercetools.api.models.shipping_method.*;
+import com.commercetools.api.models.shopping_list.*;
+import com.commercetools.api.models.staged_quote.StagedQuote;
+import com.commercetools.api.models.staged_quote.StagedQuoteSetCustomFieldAction;
+import com.commercetools.api.models.staged_quote.StagedQuoteUpdateBuilder;
+import com.commercetools.api.models.standalone_price.*;
+import com.commercetools.api.models.state.State;
+import com.commercetools.api.models.state.StateReference;
+import com.commercetools.api.models.state.StateResourceIdentifier;
+import com.commercetools.api.models.store.*;
+import com.commercetools.api.models.subscription.Subscription;
+import com.commercetools.api.models.subscription.SubscriptionSetChangesAction;
+import com.commercetools.api.models.subscription.SubscriptionSetMessagesAction;
+import com.commercetools.api.models.subscription.SubscriptionUpdateBuilder;
+import com.commercetools.api.models.tax_category.*;
+import com.commercetools.api.models.type.*;
+import com.commercetools.api.models.zone.Zone;
+import com.commercetools.api.models.zone.ZoneReference;
+import com.commercetools.api.models.zone.ZoneResourceIdentifier;
+
+import io.vrap.rmf.base.client.utils.json.JsonUtils;
+
+import org.javamoney.moneta.CurrencyUnitBuilder;
+import org.javamoney.moneta.FastMoney;
+
+// the scope of this class is to give the possibility to the JAvaDoc to get the related examples
+public class HelperMethodsTest {
+    ProjectApiRoot projectApiRoot;
+
+    // Business Unit helper methods examples
+    BusinessUnit businessUnit;
+
+    public void businessUnitSetCustomFieldUnset() {
+        final BusinessUnit updatedBusinessUnit = projectApiRoot.businessUnits()
+                .withId(businessUnit.getId())
+                .post(BusinessUnitUpdateBuilder.of()
+                        .version(businessUnit.getVersion())
+                        .actions(BusinessUnitSetCustomFieldAction.ofUnset("test-custom-field"))
+                        .build())
+                .executeBlocking()
+                .getBody();
+    }
+
+    public void businessUnitSetAddressCustomFieldUnset() {
+        final BusinessUnit updatedBusinessUnit = projectApiRoot.businessUnits()
+                .withId(businessUnit.getId())
+                .post(BusinessUnitUpdateBuilder.of()
+                        .version(businessUnit.getVersion())
+                        .actions(BusinessUnitSetAddressCustomFieldAction.ofUnset("test-address-custom-field"))
+                        .build())
+                .executeBlocking()
+                .getBody();
+    }
+
+    //Cart helper methods examples
+    Cart cart;
+
+    // TODO to be added
+    public void cartInventoryModeDefaultValue() {
```

Comment:
already opened a task in  the code generator to fix this issue https://github.com/commercetools/rmf-codegen/issues/298